src/main/java/frc/robot/Robot.java

```diff
@@ -87,4 +91,52 @@ public void simulationPeriodic() { }
 	// Do not use robotPeriodic. Use mode specific periodic methods instead.
 	@Override
 	public void robotPeriodic() { }
+
+	/** This method is for depositing high in game.
+	* @return completion of the deposit
+ 	*/
+	public boolean depositHigh() {
```

Comment:
You should not be adding functions to Robot.java. Subsystem code should be contained within your subsystem.